express-LocalLibrary/routes/status.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET Status. */
+router.get('/', function(req, res, next) {
+    res.render('status', { status: 'Online'});
+  });
+
+module.exports = router;
+
+/* 
+Coded by Jianting
+*/
```

Comment:
@clarkngo  do you require this? If not, it's not needed. We can look at the git history to determine who coded what.

Reply:
Got it.